### PR TITLE
Unity master staging aot profile

### PR DESCRIPTION
**Pull request by mrvoorhe**
Changes for building the unity_aot profile.

### CODE REVIEW

File: mcs/build/profiles/mobile_static.make [hunk 1]
```diff
@@ -40,8 +40,12 @@ MOBILE_STATIC = yes
...
 
+# Unity : disable signing
+NO_SIGN_ASSEMBLY = yes
+
 # Note need for trailing comma. If you add, keep it
 PROFILE_TEST_HARNESS_EXCLUDES = MobileNotWorking,PKITS,
 
-ALWAYS_AOT = yes
+# Unity : don't generate aot assemblies as part of the profile build
+#ALWAYS_AOT = yes
```
Comment (joncham) [inline mcs/build/profiles/mobile_static.make]: I am not sure we should be modifying upstream files here vs creating our own profile. Maybe fine for now...

Comment (joshpeterson) [inline mcs/build/profiles/mobile_static.make]: Ok, makes sense to me.

**Comment by joshpeterson** [comment]
LGTM

**Comment by joncham** [comment]
LGTM

**Pull request merged**